<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Analytics</class>
 <widget class="QMainWindow" name="Analytics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI Emoji</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Analytics</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(212, 108, 78);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"> QScrollBar:vertical {
	border: none;
	background: rgb(249, 204, 158);
    width: 14px;
    margin: 15px 0 15px 0;
	border-radius: 0px;
 }

/*  HANDLE BAR VERTICAL */
QScrollBar::handle:vertical {	
	background-color: rgb(255, 129, 94);
	min-height: 30px;
	border-radius: 7px;
}
QScrollBar::handle:vertical:hover{	
	background-color: rgb(221, 154, 109);
}
QScrollBar::handle:vertical:pressed {	
	background-color: rgb(212, 108, 78);
}

/* BTN TOP - SCROLLBAR */
QScrollBar::sub-line:vertical {
	border: none;
	background-color: rgb(89, 52, 27);
	height: 15px;
	border-top-left-radius: 7px;
	border-top-right-radius: 7px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical:hover {	
	background-color: rgb(197, 114, 59);
}
QScrollBar::sub-line:vertical:pressed {	
	background-color: rgb(49, 28, 15);
}

/* BTN BOTTOM - SCROLLBAR */
QScrollBar::add-line:vertical {
	border: none;
	background-color: rgb(89, 52, 27);
	height: 15px;
	border-bottom-left-radius: 7px;
	border-bottom-right-radius: 7px;
	subcontrol-position: bottom;
	subcontrol-origin: margin;
}
QScrollBar::add-line:vertical:hover {	
	background-color: rgb(197, 114, 59);
}
QScrollBar::add-line:vertical:pressed {	
	background-color: rgb(49, 28, 15);
}


QPushButton{
	background-color: rgb(249, 204, 158);
	color: rgb(89, 52, 27);
	border-radius: 10px;
}
QPushButton::hover{
	border: 2px solid rgb(138, 80, 42);
}
QPushButton::pressed{
	border: 2px solid rgb(89, 52, 27);
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="75,25">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(250, 240, 202);
border-radius: 10px
</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(250, 240, 202);
border-radius: 10px
</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="SalesLabel">
            <property name="font">
             <font>
              <family>Amsterdam Traveling</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(250, 240, 202);
border-radius: 10px
</string>
            </property>
            <property name="text">
             <string>Sales</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QGridLayout" name="gridLayout_2" rowstretch="60,5,35" columnstretch="60,0">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item row="2" column="1">
           <widget class="QTableWidget" name="SummaryStatistics">
            <property name="styleSheet">
             <string notr="true">QTableWidget{
color: rgb(0, 0, 0);
border-radius: 10px;
background-color: rgb(249, 204, 158);
border-radius: 10px;
font:  12px 'Georgia';
}

QHeaderView::section{
	background-color: rgb(250, 240, 202);
	border: 1px solid  rgb(249, 204, 158);
	font: 14px 'Georgia';
}
</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QTreeView" name="OrderQuantitiesList">
            <property name="styleSheet">
             <string notr="true">

QTreeView{
color: rgb(0, 0, 0);
border-radius: 10px;
	background-color: rgb(250, 240, 202);


border-radius: 10px;
font:  12px 'Georgia';
}

QTreeView::item{

background-color: rgb(249, 204, 158);
	color: rgb(89, 52, 27);
}
QTreeView::item:hover {
background-color: rgb(250, 240, 202);

}

QTreeView::item:selected {
	background-color: rgb(89, 52, 27);

	color: rgb(250, 240, 202);
	font-weight:bold;
}

QHeaderView::section{
	background-color: rgb(250, 240, 202);
	border: 1px solid  rgb(249, 204, 158);
	font: 14px 'Georgia';
}</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="OrderQuantitiesLabel">
            <property name="font">
             <font>
              <family>Amsterdam Traveling</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(250, 240, 202);
border-radius: 10px
</string>
            </property>
            <property name="text">
             <string>Product Summary</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="SummaryList">
            <property name="font">
             <font>
              <family>Amsterdam Traveling</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(250, 240, 202);
border-radius: 10px
</string>
            </property>
            <property name="text">
             <string>Statistical Summary</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QCustomPlot" name="Graph" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
border-radius: 10px;
background-color: rgb(249, 204, 158);
</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(250, 240, 202);
border-radius: 10px
</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="CloseButton">
         <property name="font">
          <font>
           <family>Georgia</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(249, 204, 158);
color: rgb(89, 52, 27);
border-radius: 10px;
padding: 3px 3px 3px 3px;</string>
         </property>
         <property name="text">
          <string>Close Analytics</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="RestaurantLogoLabel">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/logo/pictures/logo 2.jpg);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="resources.qrc">:/logo/pictures/logo 3.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="DateSelectionLabel">
         <property name="font">
          <font>
           <family>Amsterdam Traveling</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(250, 240, 202);
border-radius: 7px
</string>
         </property>
         <property name="text">
          <string>Date Selection</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(250, 240, 202);
border-radius: 10px
</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="topMargin">
           <number>1</number>
          </property>
          <item row="1" column="0" colspan="2" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="StartDate">
            <property name="font">
             <font>
              <family>Georgia</family>
              <pointsize>11</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);background-color: rgb(250, 240, 202);</string>
            </property>
            <property name="text">
             <string>(start date)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="StartDateButton">
            <property name="font">
             <font>
              <family>Georgia</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(249, 204, 158);
color: rgb(89, 52, 27);
border-radius: 10px;
padding: 3px 3px 3px 3px;</string>
            </property>
            <property name="text">
             <string>Select Start Date</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="EndDateLabel">
            <property name="font">
             <font>
              <family>Georgia</family>
              <pointsize>11</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>End Date</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="EndDateButton">
            <property name="font">
             <font>
              <family>Georgia</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(249, 204, 158);
color: rgb(89, 52, 27);
border-radius: 10px;
padding: 3px 3px 3px 3px;</string>
            </property>
            <property name="text">
             <string>Select End Date</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="EndDate">
            <property name="font">
             <font>
              <family>Georgia</family>
              <pointsize>11</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>(end date)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="StartDateLabel">
            <property name="font">
             <font>
              <family>Georgia</family>
              <pointsize>11</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);background-color: rgb(250, 240, 202);</string>
            </property>
            <property name="text">
             <string>Start Date:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="TransactionsListLabel">
         <property name="font">
          <font>
           <family>Amsterdam Traveling</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(250, 240, 202);
border-radius: 10px
</string>
         </property>
         <property name="text">
          <string>List of Transaction IDs:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="TransactionsList">
         <property name="styleSheet">
          <string notr="true">QListView {
    show-decoration-selected: 0; /* make the selection span the entire width of the view */
	
	background-color: rgb(249, 204, 158);
	border-radius: 10px;
	font:  12px 'Georgia';
	padding: 2px 2px 2px 2px;
}
QListView::item{
	padding: 1px 0px 1px 0px;
}
QListView::item:selected {
	background-color: rgb(89, 52, 27);
	color: rgb(250, 240, 202);
}

QListView::item:selected:active {
 	background-color: rgb(89, 52, 27);
	color: rgb(250, 240, 202);
}

QListView::item:hover {
	background-color: rgb(250, 240, 202);
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ViewTransactionButton">
         <property name="font">
          <font>
           <family>Georgia</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(249, 204, 158);
color: rgb(89, 52, 27);
border-radius: 10px;
padding: 3px 3px 3px 3px;</string>
         </property>
         <property name="text">
          <string>View Selected Transaction</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="UpdateButton">
         <property name="font">
          <font>
           <family>Georgia</family>
           <pointsize>14</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(249, 204, 158);
color: rgb(89, 52, 27);
border-radius: 10px;
padding: 3px 3px 3px 3px;</string>
         </property>
         <property name="text">
          <string>Update Selection</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
